src/components/swap/SwapModalFooter.tsx

```diff
@@ -209,3 +217,36 @@ export default function SwapModalFooter({
     </>
   )
 }
+
+function TokenTaxLineItem({ trade, type }: { trade: ClassicTrade; type: 'input' | 'output' }) {
```

Comment:
I think there should be a higher-level refactor to share components between the swap details in the dropdown & the review modal details, whereas sharing a component for just these tax line items would be wonky since they have different styling